app/src/main/java/org/simple/clinic/patient/Patient.kt

```diff
@@ -217,6 +217,13 @@ data class Patient(
         scheduled: Appointment.Status = Appointment.Status.SCHEDULED
     ): Flowable<Boolean>
 
+    @Query("""
+      UPDATE Patient
+      SET updatedAt = :newUpdatedAt, syncStatus = :newSyncStatus
+      WHERE uuid = :patientUuid
+    """)
+    abstract fun touch(patientUuid: UUID, newUpdatedAt: Instant, newSyncStatus: SyncStatus)
```

Comment:
We do not need to query since we only need to update the `syncStatus`.
There is already a method for that called `updateSyncStatus`.

Reply:
Done.